src/fosslight_source/cli.py

```
@@ -394,7 +394,7 @@ def _collect_kb_file_hashes(
 
 
 def merge_results(
-    scancode_result: list = [], scanoss_result: list = [], spdx_downloads: dict = {},
+    scancode_result: list = [], scanoss_result: list = [], spdx_downloads: dict = {}, spdx_licenses: dict = {},
```

Ruff (0.16.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)